cli/src/build-settings.ts

```diff
@@ -13,35 +13,46 @@ function isStringDict(x: JSON.Object): x is Dict<string | null> {
 
 export default class BuildSettings {
   private rustc: string;
+  private nodeVersion: string;
```

Comment:
Based on my comment about defaulting below, this field will need to be `string | null`.

Reply:
It's worth adding a comment in the code here, explaining that it can be `null` because older versions of Neon didn't have this key. That will help the null case be understandable/maintainable.